ui/src/SpaceDashboard/SpaceDashboard.tsx

```diff
@@ -107,12 +107,14 @@ function SpaceDashboard({
 
     function NewSpaceButton(): JSX.Element {
         return (
-            <button className="createNewSpaceButton" onClick={onCreateNewSpaceButtonClicked}>
-                <i className="material-icons createNewSpaceIcon">
-                    add_circle_outline
-                </i>
-                Create New Space
-            </button>
+            <div>
```

Comment:
What was the intention of wrapping this in a div? I'm curious.

Reply:
I believe the idea was to bring consistency to the new space button and the space tiles - @dctid, you had more of an opinion on this than I when we were pairing; I don't want to misrepresent your reasoning